toolchain/sem_ir/name_scope.h

```diff
@@ -72,40 +153,25 @@ struct NameScope : Printable<NameScope> {
   // vector so that these can pack densely. If this ends up both cold and memory
   // intensive, we can also switch the lookup to a set of indices into the
   // vector rather than a map from `NameId` to index.
-  llvm::SmallVector<Entry> names;
-  Map<NameId, int> name_map;
+  llvm::SmallVector<Entry> names_;
+  Map<NameId, int> name_map_;
 
-  // Instructions returning values that are extended by this scope.
-  //
   // Small vector size is set to 1: we expect that there will rarely be more
   // than a single extended scope.
   // TODO: Revisit this once we have more kinds of extended scope and data.
   // TODO: Consider using something like `TinyPtrVector` for this.
-  llvm::SmallVector<InstId, 1> extended_scopes;
```

Comment:
What do you think about moving these comments back to the members? When we have accessors, we tend to keep comments on the member itself (typically no comment on trivial accessors/mutators). I'd lean towards that here, too. It means that regardless of whether a developer is aware that there's an accessor, they should be able to look at the data member to see an explanation of what it is. I think this can also help make clear that, when there are details about the member, those aren't the _only_ comments for the member.

Reply:
Moved the trivial accessors comments to the data members.
My tendency is to document the public API and only keep implementation comments in the non public section or function definitions, but I see the value of having the comments in the data members.